Formula/spice-protocol.rb

```diff
@@ -36,9 +41,13 @@ def install
         return (SPICE_LINK_ERR_OK == 0) ? 0 : 1;
       }
     EOS
-    system ENV.cc, "test.cpp",
-                   "-I#{include}/spice-1",
-                   "-o", "test"
+
+    cc = ENV.cc
+    on_linux do
+      cc = Formula["gcc"].opt_bin/"gcc-#{Formula["gcc"].any_installed_version.major}"
+    end
+
```

Comment:
Just so I understand this, the formula itself builds fine with `gcc-5`, but the test binary does not?  I guess there's nothing wrong with this in principal, I've just never seen this happen before.  It must mean they don't ever include that header file when building?

Reply:
I think this is a header-only package, so it doesn't use GCC during build.

We could make it `fails_with gcc: "5"`; though, it would add an unnecessary brewed GCC dependency for build-time.

Reply:
Ah, that makes sense.  Anything which depends on it will have to use brewed GCC of course, but there's no reason to drag it in here.